lib/colorls/core.rb

```diff
@@ -298,6 +304,12 @@ def git_dir_info(content, status)
       end
     end
 
+    def inode(content)
+      return '' unless @show_inode
+
+      content.stats.ino.to_s.colorize(@colors[:inode])
+    end
```

Comment:
The returned string should be right justified to make the numbers line up, and ensure that every inode number occupies the same amount of space:
```
   4849224     doc/                        3158102     fooooo 
  3771969     empty/                      6077571     Gemfile 
  422416     exe/                        3226807     Gemfile.lock 
  4476968     gg/                         461157     ISSUE_TEMPLATE.md 
```